tools/rosbag/README.md

```diff
@@ -0,0 +1,32 @@
+# rosbag
+This is a custom rosbag package which provides all the feature of [original rosbag](http://wiki.ros.org/rosbag/Commandline) with custom freq recording of topics.
```

Comment:
This doesn't sounds right. This is the readme in the `rosbag` directory - its not a "custom rosbag package".